data/korath/korath missions.txt

```diff
@@ -257,3 +257,223 @@ mission "Small Scale Delivery"
+			`	"Salutations Captain of the <ship>. Your most diligent efforts are required lest many families mourn the dead. An Efreti ship, the <EDC-ship>, was surveying <stopovers> when surprised by Mereti they were. Their ship crashed on the planet with many dead and they have asked for help to evacuate."`
+			choice
+				`	"I'll go there immediately."`
+					accept
+				`	"Can't the Quarg respond?"`
+					goto quarg
+				`	"I don't care about the Korath."`
+					goto careless
+			
+			label quarg
+			`	"We fought off the Mereti that attacked them but they had crashed before we could reach them. The gravity on that world requires us to have hardened environment suits to navigate wreckage while a Human would have few problems. You can reach them days before we could return from Kuwaru Efreti."`
+			choice
+				`	"I'll go help, then."`
+					accept
+				`	"I won't risk my ship for the Korath."`
+					goto careless
+
+			label careless
+			`	The Quarg is silent for some time. Eventually it responds, "Life is rare and precious across the vastness of space. Look about the Korath worlds and see what remains when one cares not what others suffer."`
+				decline
+
+	on stopover
+		conversation
+			`It takes you some time to find the <EDC-ship>. You had expected a beacon of some sort but they are either keeping radio silence to avoid drawing the Mereti or there were no survivors.`
+			`	When you manage to open the hatch, you find yourself staring down a thermal rifle. The Korath shouts something at you and you hold your hands out palms forward. He limps back a few steps but keeps the weapon trained on you. You slowly pull your communicator out and play your ship's recording of the Quarg transmission. When the Korath hears the Quarg's voice they become excited. "Laki Nemparu?" he asks repeatedly. You nod your head and show your palms again.`
+			`	He leads you into the ship, shouting into a communication device that is then picked up on speakers throughout the ship. You can only glance around as you move through the engineering section but it's clear they are running on emergency power and their Systems Core was destroyed leaving them stranded.`
+			`	The living area is a makeshift hospital. You don't see any bodies, but there are far fewer Korath here than you would expect to crew a ship of this size. It's clear that all the surviving crew are injured to some degree with more than half incapacitated or unconscious. Even if if the ship is spaceworthy, there aren't enough crew to fly it.`
+			`	You quickly count the number of surviors and communicate to your guide that you can take them all. He mimics your nodding head gesture and shows you his plams and makes quick cirlcles. You think he is agreeing and wants to hurry. You call your crew over to help move the survivors to the <ship>. Within a couple of hours you are ready to go.`
+	
+	on complete
+		payment 3000000
```

Comment:
Is this payment from the Efreti or Quarg? Seems a little large for the Efreti - I'm wondering if they could afford that. I wrote previous missions characterizing the Efreti as insistent on paying you, even if it wasn't a lot (see Korath Family to Ringworld mission). If that characterization still exists in-game, think that's something that makes sense to touch on here?

Reply:
There's something of a trade-off here. If we pay the player too little, then it pushes the player towards capture/salvage.

On the other hand, payment is an important story tool for conveying information to the player, just like other things.

3M seems a bit high for the way Efreti have been portrayed in the past, so I'd suggest dropping it. Perhaps about to about 1M? 

That being said, splitting it up so that the Efreti rescued here pay you some about, maybe even less than 1M - but then the Quarg quietly "top it up" to something more significant, perhaps even up to the 3M listed here shortly afterwards. Maybe a day or two later. This both contributes to the portrayal of the Efreti as both wanting to pay and unable to pay very much, while reinforcing the Quarg's interesting in rewarding the protagonist for friendly / aiding behavior, which seems both in keeping for their role in protecting the Efreti and their overall interest in promoting peace.

Reply:
A more interesting alternative is to "pay" the player with ship outfits. That can only work if the payment is made at a planet with an outfitter. Otherwise, if it ends up in the planetary storage, you have no way to get it back.